python/FluidSolver.py

```diff
@@ -109,15 +109,17 @@
     if interface.is_action_required(action_write_iteration_checkpoint()):
         interface.fulfilled_action(action_write_iteration_checkpoint())
 
-    velocity, pressure, success = perform_partitioned_implicit_euler_step(velocity_n, pressure_n, crossSectionLength_n, crossSectionLength, dx, precice_tau, config.velocity_in(t + precice_tau), custom_coupling=False)
+    velocity, pressure, success = perform_partitioned_implicit_euler_step(velocity_n, pressure_n, crossSectionLength_n,
+                                                                          crossSectionLength, dx, precice_dt, config.velocity_in(t + precice_dt), custom_coupling=False)
     interface.write_block_scalar_data(pressureID, vertexIDs, pressure)
-    precice_tau = interface.advance(precice_tau)
+    dt = interface.advance(precice_dt)
+    precice_dt = min(precice_dt, dt)
```

Comment:
We don't need the `min`, since we only have a single timestep size `precice_dt` that is matching for precice and the solver (see above).

Reply:
Just write `precice_dt = interface.advance(precice_dt)`